modules/core/src/experimental/reflection-effect/reflection-effect.js

```diff
@@ -21,7 +21,7 @@
 /* global window */
 import GL from '@luma.gl/constants';
 import {Framebuffer, Model, Geometry} from 'luma.gl';
-import Effect from '../lib/effect';
+import Effect from '../../effects/effect';
```

Comment:
I personally like having the base class in the `lib` (I consider it part of the "machinery"). And I like to regard the `effects` folder as an "independent" catalog of usable effects built on the base Effect class in the lib. This would parallel `Layer` and `CompositeLayer` being in the lib but the layer subclasses being in `layers`.